harvester_e2e_tests/apis/test_networks.py

```diff
@@ -58,16 +58,30 @@ def test_create_without_name(self, api_client):
         assert 422 == code, (code, data)
         assert "Invalid" == data.get("reason"), (code, data)
 
+    @pytest.mark.parametrize("vlan_id", [4095])
+    @pytest.mark.skip_version_before("v1.1.0")  # ref to harvester/issues/3151
+    def test_create_with_invalid_id_110(self, api_client, unique_name, vlan_id):
+        code, data = api_client.networks.create(unique_name, vlan_id)
+
+        assert 500 == code, (vlan_id, code, data)
+        assert "InternalError" == data.get("reason"), (vlan_id, code, data)
+
 
 @pytest.mark.p0
 @pytest.mark.networks
 class TestNetworks:
 
     @pytest.mark.dependency(name="create_network")
+    @pytest.mark.skip_version_after("v1.0.3")
     def test_create(self, api_client, unique_name):
         code, data = api_client.networks.create(unique_name, VLAN_ID)
         assert 201 == code, (code, data)
 
+    @pytest.mark.skip_version_before("v1.1.0")
+    def test_create_110(self, api_client, unique_name):
```

Comment:
Same as above comment, let's keep the name as `test_create` of the test which will run after v1.1.0 and `test_create_103` for v1.0.3

Reply:
updated